spk/domoticz/src/dsm-control.sh

```diff
@@ -16,6 +16,14 @@ PORT="8084"
 
 start_daemon ()
 {
+    insmod /lib/modules/usbserial.ko
+    insmod /lib/modules/ftdi_sio.ko
+    insmod ${INSTALL_DIR}/modules/cp210x.ko
```

Comment:
I originally had that, but it occurred to me, especially with my other pull request for OctoPrint, that removing USB drivers because I'm done with them isn't really very polite.